### ISSUE TITLE
Mermaid diagram rendering broken

### ISSUE THREAD

**Issue by jemgunay** [colinwilson/lotusdocs#138]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for support

### Describe the bug

I attempted to embed Mermaid diagrams in a markdown page, but the built site fails to render them correctly. They seem to successfully render locally when running via `hugo server`, but not when built via `hugo`. I also see Javascript errors related to Mermaid in the browser console.

This issue appears to exist on the published lotusdocs documentation also: https://lotusdocs.dev/docs/features/mermaid/

### Expected behavior

The built site to correctly render Mermaid diagrams defined in markdown.

### Screenshots

Example of failed rendering & console error on lotus docs page:
![Screenshot 2023-11-27 at 15 07 24](https://github.com/colinwilson/lotusdocs/assets/5686734/30cb9519-c237-46bb-904d-57885cbe0a97)

Location of console error:
![Screenshot 2023-11-27 at 15 09 21](https://github.com/colinwilson/lotusdocs/assets/5686734/a809d2cb-a2d7-4fa1-8b48-cc1892aed453)

### Environment

```markdown
- OS: MacOS M2 & Android mobile
- Browser: Chrome & Firefox on desktop, Chrome on mobile
- Browser Version: Chrome Version 119.0.6045.159 (Official Build) (arm64)
```

**Comment by colinwilson** [comment]
Thanks for catching this @jemgunay 👍. Will issue a fix very soon.

**Comment by rusinikita** [comment]
A bit of context. Locally it runs good.

[Page with no render](https://rusinikita.github.io/changes/docs/adr/23.10-init/)

Local render. [Setup](https://github.com/rusinikita/changes/tree/main/docs)
<img width="1167" alt="Screenshot 2023-11-29 at 17 51 18" src="https://github.com/colinwilson/lotusdocs/assets/10090151/3e3cc587-74ba-427d-bcf1-ad6aa5d5511d">

**Comment by colinwilson** [comment]
Thanks. 👍

I think the issue is with the minifcation of the mermaid JavaScript.

**Comment by colinwilson** [comment]
Fixed by #141

**Comment by jemgunay** [comment]
@colinwilson Rebuilt with the latest release commit and works as expected. Swift fix much appreciated 🙏🏽
